lib/util/state.js

```diff
@@ -119,16 +119,32 @@ export const getActiveItineraries = createSelector(
   }
 )
 
-/**
- * If there are no itineraries, generate an error
- * so we're not left with an empty results container.
- */
-export const createNoItinerariesErr = (state, errors, pending) => {
+export function getActiveSearchErrors(state) {
+  const activeSearch = getActiveSearch(state)
+  if (!activeSearch) return {}
+  const routingErrors = activeSearch?.response?.reduce((acc, cur) => {
```

Comment:
Maybe rename the top-level `routingErrors` to something else to not confuse with the inner loop veriable below.